step-definitions/standard-ui/form.js

```diff
@@ -179,3 +203,18 @@ Then('Input fields with labels {string} should be visible',
     client.useCss();
   }
 )
+
+/**
+ * Click on label next to the evan checkbox control
+ */
+Then('I want to click on vue checkbox control with id {string}',
```

Comment:
it actually clicks the checkbox, so I would delete the `want to` part of this string.